src/blocks/modal.css

```diff
@@ -40,6 +40,7 @@
 }
 
```

Comment:
* The `avatar` and `confirmation` texts have incorrect `font`. Please, add `font-family: "Poppins", Arial, Helvetica, sans-serif;` inside `modal.css` into line 9 for the `.modal` class


```css

.modal {
  font-family: "Poppins", Arial, Helvetica, sans-serif;
  position: fixed;
// the rest of the styles
```